### PR TITLE
change content

### PR DESCRIPTION
**Pull request by stevelee-civica**
add conditional render

### CODE REVIEW

File: src/EA.Weee.Web/Controllers/OrganisationRegistrationController.cs [hunk 1]
```diff
@@ -689,6 +689,8 @@ public async Task<ActionResult> ContactDetails()
             var countries = await GetCountries();
             model.AddressData.Countries = countries;
 
+            model.HasAuthorisedRepresentitive = existingTransaction?.AuthorisedRepresentative == YesNoType.Yes;
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Controllers/OrganisationRegistrationController.cs:692]: @stevelee-civica  can you create a unit tests to ensure this value is set correctly...

File: src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerSubmissionControllerTests.cs [hunk 2]
```diff
@@ -261,7 +261,7 @@ public async Task EditContactDetails_Post_ValidModel_ShouldRedirectToTaskList()
 
             // Assert
             result.Should().NotBeNull();
-            result.RouteValues["action"].Should().Be("TaskList");
+            result.RouteValues["action"].Should().Be("CheckAnswers");
```
Comment (will-hope98) [inline src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerSubmissionControllerTests.cs:264]: @stevelee-civica does this test pass now?  I'm not sure this change is needed

Comment (stevelee-civica) [reply src/EA.Weee.Web.Tests.Unit/Areas/Producer/Controllers/ProducerSubmissionControllerTests.cs]: restored the changes
it fails the test case now

File: src/EA.Weee.Web/Controllers/OrganisationRegistrationController.cs (hunk 1 above)
Comment (stevelee-civica) [reply src/EA.Weee.Web/Controllers/OrganisationRegistrationController.cs:741]: updated and added assertions on without existing transaction

**Pull request merged**